modules/videoio/src/cap_v4l.cpp

```diff
@@ -285,149 +283,161 @@ struct CvCaptureCAM_V4L CV_FINAL : public CvCapture
     bool convert_rgb;
     bool frame_allocated;
     bool returnFrame;
+    // To select a video input set cv::CAP_PROP_MODE to channel number.
+    // If the new channel number is greater than 32768 or less than 0, then a video input will not change
+    int channelNumber;
+    // Backward compatibility for parameters. If set parameters will be converted to/from [0,1) range.
+    // To select real parameters mode set cv::CAP_PROP_MODE to 65536
+    // any other value greater than 32768 revert backward compatibility mode.
+    // Backward compatibility mode affects the following parameters:
+    // cv::CAP_PROP_*: BRIGHTNESS,CONTRAST,SATURATION,HUE,GAIN,EXPOSURE,FOCUS,AUTOFOCUS,AUTO_EXPOSURE.
+    bool compatibilityMode;
```

Comment:
Comment is outdated.
`CAP_PROP_MODE` is already used, so behavior should not be changed.

Alternative variant via environment variables:
```
compatibilityMode = !utils::getConfigurationParameterBool("OPENCV_VIDEOIO_V4L_RANGE_NORMALIZED", false);
```

Reply:
`CAP_PROP_MODE` was used for get - not for set